extensions/interactions/GraphInput/directives/graph-utils.service.ts

```diff
@@ -48,7 +50,7 @@ export class GraphUtilsService {
    */
   constructAdjacencyLists(
       graph: GraphAnswer, adjacencyListMode: string): number[][] {
-    var adjacencyLists = [];
+    var adjacencyLists: number[][] = [];
```

Comment:
Should this be `(number | null)` instead?

Reply:
I don't think so, because an adjacency list cannot contain null elements. Each sub-array is grown when a new node is added on to the end.

Reply:
@iamprayush Could you take a look at this when you get a chance?

Reply:
Got it, thanks for explaining!